src/components/ContentHeader.vue

```diff
@@ -15,7 +15,7 @@
           <ul class="usa-nav__primary usa-accordion">
             <li class="usa-nav__primary-item">
               <button
-                class="usa-nav__button usa-nav__link"
+                class="usa-nav__button usa-nav__link first-button"
                 aria-expanded="false"
                 onClick="document.getElementById('viz-new').scrollIntoView();"
               >
```

Comment:
Another not urgent change -- One note about these Nav buttons - right now, Recent Work and Visualizations take you to a view where you can't see those headers, and see just the content, while About takes you so that you see half the header:
![image](https://user-images.githubusercontent.com/54007288/140169345-3c417df3-84d0-46a5-807d-9b9d3a2f3756.png)
That last is I think b/c there isn't enough room below it on the page. 

I think this would ideally be consistent, and my personal preference would be to have the header visible for all of them

Reply:
Yeah! That is on my to-do list to address, couldn't figure it out yet.